{"uid":"a8d23136adcbda7d","name":"Card Creation","fullName":"tests.regression_ui.test_ui_regression.TestUIRegression#test_card_creation","historyId":"37b04428e9c82a14933f1043cf15ed79","time":{"start":1734113658870,"stop":1734113696685,"duration":37815},"status":"broken","statusMessage":"TypeError: not all arguments converted during string formatting","statusTrace":"self = <tests.regression_ui.test_ui_regression.TestUIRegression object at 0x7f6098e18c40>\ndriver = <selenium.webdriver.chrome.webdriver.WebDriver (session=\"f397d1f36411738e91666fc5856585b1\")>\n\n    @pytest.mark.TC000\n    @pytest.mark.TC004\n    @allure.title('Card Creation')\n    def test_card_creation(self, driver):\n        time.sleep(10)\n        with allure.step('Log in to Trello account.'):\n            self.test_login(driver)\n        with allure.step('Open a board with an existing list and create a new card'):\n            try:\n>               WebDriverWait(driver, 10).until(EC.element_to_be_clickable((By.XPATH, self.SCC.Board.BOARD_TITLE))).click()\n\ntests/regression_ui/test_ui_regression.py:146: \n_ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ \n\nself = <selenium.webdriver.support.wait.WebDriverWait (session=\"f397d1f36411738e91666fc5856585b1\")>\nmethod = <function element_to_be_clickable.<locals>._predicate at 0x7f6097ed0e50>\nmessage = ''\n\n    def until(self, method: Callable[[D], Union[Literal[False], T]], message: str = \"\") -> T:\n        \"\"\"Calls the method provided with the driver as an argument until the \\\n        return value does not evaluate to ``False``.\n    \n        :param method: callable(WebDriver)\n        :param message: optional message for :exc:`TimeoutException`\n        :returns: the result of the last call to `method`\n        :raises: :exc:`selenium.common.exceptions.TimeoutException` if timeout occurs\n        \"\"\"\n        screen = None\n        stacktrace = None\n    \n        end_time = time.monotonic() + self._timeout\n        while True:\n            try:\n                value = method(self._driver)\n                if value:\n                    return value\n            except self._ignored_exceptions as exc:\n                screen = getattr(exc, \"screen\", None)\n                stacktrace = getattr(exc, \"stacktrace\", None)\n            if time.monotonic() > end_time:\n                break\n            time.sleep(self._poll)\n>       raise TimeoutException(message, screen, stacktrace)\nE       selenium.common.exceptions.TimeoutException: Message: \nE       Stacktrace:\nE       #0 0x55caee44c31a <unknown>\nE       #1 0x55caedf626e0 <unknown>\nE       #2 0x55caedfb13e6 <unknown>\nE       #3 0x55caedfb1681 <unknown>\nE       #4 0x55caedff6b04 <unknown>\nE       #5 0x55caedfd548d <unknown>\nE       #6 0x55caedff3ed7 <unknown>\nE       #7 0x55caedfd5203 <unknown>\nE       #8 0x55caedfa3cc0 <unknown>\nE       #9 0x55caedfa4c9e <unknown>\nE       #10 0x55caee419d0b <unknown>\nE       #11 0x55caee41dc92 <unknown>\nE       #12 0x55caee406b3c <unknown>\nE       #13 0x55caee41e807 <unknown>\nE       #14 0x55caee3ec0df <unknown>\nE       #15 0x55caee43b578 <unknown>\nE       #16 0x55caee43b740 <unknown>\nE       #17 0x55caee44b196 <unknown>\nE       #18 0x7f6346c94ac3 <unknown>\n\n/opt/hostedtoolcache/Python/3.8.18/x64/lib/python3.8/site-packages/selenium/webdriver/support/wait.py:105: TimeoutException\n\nDuring handling of the above exception, another exception occurred:\n\nself = <tests.regression_ui.test_ui_regression.TestUIRegression object at 0x7f6098e18c40>\ndriver = <selenium.webdriver.chrome.webdriver.WebDriver (session=\"f397d1f36411738e91666fc5856585b1\")>\n\n    @pytest.mark.TC000\n    @pytest.mark.TC004\n    @allure.title('Card Creation')\n    def test_card_creation(self, driver):\n        time.sleep(10)\n        with allure.step('Log in to Trello account.'):\n            self.test_login(driver)\n        with allure.step('Open a board with an existing list and create a new card'):\n            try:\n                WebDriverWait(driver, 10).until(EC.element_to_be_clickable((By.XPATH, self.SCC.Board.BOARD_TITLE))).click()\n                WebDriverWait(driver, 10).until(EC.visibility_of_element_located((By.XPATH, self.SCC.Board.LIST_TITLE)))\n                driver.find_element(By.XPATH, self.SCC.List.ADD_A_CARD_BUTTON).click()\n                driver.find_element(By.XPATH, self.SCC.List.CARD_NAME_FIELD).click()\n                time.sleep(1)\n                driver.find_element(By.XPATH, self.SCC.List.CARD_NAME_FIELD).send_keys('new_card')\n                driver.find_element(By.XPATH, self.SCC.List.ADD_CARD_SUBMIT_BUTTOMN).click()\n            except Exception as e:\n>               log.info(\"An error occurred: \", str(e))\n\ntests/regression_ui/test_ui_regression.py:154: \n_ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ \n/opt/hostedtoolcache/Python/3.8.18/x64/lib/python3.8/logging/__init__.py:2082: in info\n    root.info(msg, *args, **kwargs)\n/opt/hostedtoolcache/Python/3.8.18/x64/lib/python3.8/logging/__init__.py:1446: in info\n    self._log(INFO, msg, args, **kwargs)\n/opt/hostedtoolcache/Python/3.8.18/x64/lib/python3.8/logging/__init__.py:1589: in _log\n    self.handle(record)\n/opt/hostedtoolcache/Python/3.8.18/x64/lib/python3.8/logging/__init__.py:1599: in handle\n    self.callHandlers(record)\n/opt/hostedtoolcache/Python/3.8.18/x64/lib/python3.8/logging/__init__.py:1661: in callHandlers\n    hdlr.handle(record)\n/opt/hostedtoolcache/Python/3.8.18/x64/lib/python3.8/logging/__init__.py:954: in handle\n    self.emit(record)\n/opt/hostedtoolcache/Python/3.8.18/x64/lib/python3.8/site-packages/_pytest/logging.py:386: in emit\n    super().emit(record)\n/opt/hostedtoolcache/Python/3.8.18/x64/lib/python3.8/logging/__init__.py:1093: in emit\n    self.handleError(record)\n/opt/hostedtoolcache/Python/3.8.18/x64/lib/python3.8/logging/__init__.py:1085: in emit\n    msg = self.format(record)\n/opt/hostedtoolcache/Python/3.8.18/x64/lib/python3.8/logging/__init__.py:929: in format\n    return fmt.format(record)\n/opt/hostedtoolcache/Python/3.8.18/x64/lib/python3.8/site-packages/_pytest/logging.py:139: in format\n    return super().format(record)\n/opt/hostedtoolcache/Python/3.8.18/x64/lib/python3.8/logging/__init__.py:668: in format\n    record.message = record.getMessage()\n_ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ \n\nself = <LogRecord: root, 20, /home/runner/work/pythonSelenium/pythonSelenium/tests/regression_ui/test_ui_regression.py, 154, \"An error occurred: \">\n\n    def getMessage(self):\n        \"\"\"\n        Return the message for this LogRecord.\n    \n        Return the message for this LogRecord after merging any user-supplied\n        arguments with the message.\n        \"\"\"\n        msg = str(self.msg)\n        if self.args:\n>           msg = msg % self.args\nE           TypeError: not all arguments converted during string formatting\n\n/opt/hostedtoolcache/Python/3.8.18/x64/lib/python3.8/logging/__init__.py:373: TypeError","flaky":false,"newFailed":false,"newBroken":true,"newPassed":false,"retriesCount":0,"retriesStatusChange":false,"beforeStages":[{"name":"driver","time":{"start":1734113658590,"stop":1734113658868,"duration":278},"status":"passed","steps":[],"attachments":[],"parameters":[],"stepsCount":0,"shouldDisplayMessage":false,"attachmentsCount":0,"hasContent":false,"attachmentStep":false},{"name":"setup","time":{"start":1734113658869,"stop":1734113658869,"duration":0},"status":"passed","steps":[],"attachments":[],"parameters":[],"stepsCount":0,"shouldDisplayMessage":false,"attachmentsCount":0,"hasContent":false,"attachmentStep":false}],"testStage":{"status":"broken","statusMessage":"TypeError: not all arguments converted during string formatting","statusTrace":"self = <tests.regression_ui.test_ui_regression.TestUIRegression object at 0x7f6098e18c40>\ndriver = <selenium.webdriver.chrome.webdriver.WebDriver (session=\"f397d1f36411738e91666fc5856585b1\")>\n\n    @pytest.mark.TC000\n    @pytest.mark.TC004\n    @allure.title('Card Creation')\n    def test_card_creation(self, driver):\n        time.sleep(10)\n        with allure.step('Log in to Trello account.'):\n            self.test_login(driver)\n        with allure.step('Open a board with an existing list and create a new card'):\n            try:\n>               WebDriverWait(driver, 10).until(EC.element_to_be_clickable((By.XPATH, self.SCC.Board.BOARD_TITLE))).click()\n\ntests/regression_ui/test_ui_regression.py:146: \n_ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ \n\nself = <selenium.webdriver.support.wait.WebDriverWait (session=\"f397d1f36411738e91666fc5856585b1\")>\nmethod = <function element_to_be_clickable.<locals>._predicate at 0x7f6097ed0e50>\nmessage = ''\n\n    def until(self, method: Callable[[D], Union[Literal[False], T]], message: str = \"\") -> T:\n        \"\"\"Calls the method provided with the driver as an argument until the \\\n        return value does not evaluate to ``False``.\n    \n        :param method: callable(WebDriver)\n        :param message: optional message for :exc:`TimeoutException`\n        :returns: the result of the last call to `method`\n        :raises: :exc:`selenium.common.exceptions.TimeoutException` if timeout occurs\n        \"\"\"\n        screen = None\n        stacktrace = None\n    \n        end_time = time.monotonic() + self._timeout\n        while True:\n            try:\n                value = method(self._driver)\n                if value:\n                    return value\n            except self._ignored_exceptions as exc:\n                screen = getattr(exc, \"screen\", None)\n                stacktrace = getattr(exc, \"stacktrace\", None)\n            if time.monotonic() > end_time:\n                break\n            time.sleep(self._poll)\n>       raise TimeoutException(message, screen, stacktrace)\nE       selenium.common.exceptions.TimeoutException: Message: \nE       Stacktrace:\nE       #0 0x55caee44c31a <unknown>\nE       #1 0x55caedf626e0 <unknown>\nE       #2 0x55caedfb13e6 <unknown>\nE       #3 0x55caedfb1681 <unknown>\nE       #4 0x55caedff6b04 <unknown>\nE       #5 0x55caedfd548d <unknown>\nE       #6 0x55caedff3ed7 <unknown>\nE       #7 0x55caedfd5203 <unknown>\nE       #8 0x55caedfa3cc0 <unknown>\nE       #9 0x55caedfa4c9e <unknown>\nE       #10 0x55caee419d0b <unknown>\nE       #11 0x55caee41dc92 <unknown>\nE       #12 0x55caee406b3c <unknown>\nE       #13 0x55caee41e807 <unknown>\nE       #14 0x55caee3ec0df <unknown>\nE       #15 0x55caee43b578 <unknown>\nE       #16 0x55caee43b740 <unknown>\nE       #17 0x55caee44b196 <unknown>\nE       #18 0x7f6346c94ac3 <unknown>\n\n/opt/hostedtoolcache/Python/3.8.18/x64/lib/python3.8/site-packages/selenium/webdriver/support/wait.py:105: TimeoutException\n\nDuring handling of the above exception, another exception occurred:\n\nself = <tests.regression_ui.test_ui_regression.TestUIRegression object at 0x7f6098e18c40>\ndriver = <selenium.webdriver.chrome.webdriver.WebDriver (session=\"f397d1f36411738e91666fc5856585b1\")>\n\n    @pytest.mark.TC000\n    @pytest.mark.TC004\n    @allure.title('Card Creation')\n    def test_card_creation(self, driver):\n        time.sleep(10)\n        with allure.step('Log in to Trello account.'):\n            self.test_login(driver)\n        with allure.step('Open a board with an existing list and create a new card'):\n            try:\n                WebDriverWait(driver, 10).until(EC.element_to_be_clickable((By.XPATH, self.SCC.Board.BOARD_TITLE))).click()\n                WebDriverWait(driver, 10).until(EC.visibility_of_element_located((By.XPATH, self.SCC.Board.LIST_TITLE)))\n                driver.find_element(By.XPATH, self.SCC.List.ADD_A_CARD_BUTTON).click()\n                driver.find_element(By.XPATH, self.SCC.List.CARD_NAME_FIELD).click()\n                time.sleep(1)\n                driver.find_element(By.XPATH, self.SCC.List.CARD_NAME_FIELD).send_keys('new_card')\n                driver.find_element(By.XPATH, self.SCC.List.ADD_CARD_SUBMIT_BUTTOMN).click()\n            except Exception as e:\n>               log.info(\"An error occurred: \", str(e))\n\ntests/regression_ui/test_ui_regression.py:154: \n_ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ \n/opt/hostedtoolcache/Python/3.8.18/x64/lib/python3.8/logging/__init__.py:2082: in info\n    root.info(msg, *args, **kwargs)\n/opt/hostedtoolcache/Python/3.8.18/x64/lib/python3.8/logging/__init__.py:1446: in info\n    self._log(INFO, msg, args, **kwargs)\n/opt/hostedtoolcache/Python/3.8.18/x64/lib/python3.8/logging/__init__.py:1589: in _log\n    self.handle(record)\n/opt/hostedtoolcache/Python/3.8.18/x64/lib/python3.8/logging/__init__.py:1599: in handle\n    self.callHandlers(record)\n/opt/hostedtoolcache/Python/3.8.18/x64/lib/python3.8/logging/__init__.py:1661: in callHandlers\n    hdlr.handle(record)\n/opt/hostedtoolcache/Python/3.8.18/x64/lib/python3.8/logging/__init__.py:954: in handle\n    self.emit(record)\n/opt/hostedtoolcache/Python/3.8.18/x64/lib/python3.8/site-packages/_pytest/logging.py:386: in emit\n    super().emit(record)\n/opt/hostedtoolcache/Python/3.8.18/x64/lib/python3.8/logging/__init__.py:1093: in emit\n    self.handleError(record)\n/opt/hostedtoolcache/Python/3.8.18/x64/lib/python3.8/logging/__init__.py:1085: in emit\n    msg = self.format(record)\n/opt/hostedtoolcache/Python/3.8.18/x64/lib/python3.8/logging/__init__.py:929: in format\n    return fmt.format(record)\n/opt/hostedtoolcache/Python/3.8.18/x64/lib/python3.8/site-packages/_pytest/logging.py:139: in format\n    return super().format(record)\n/opt/hostedtoolcache/Python/3.8.18/x64/lib/python3.8/logging/__init__.py:668: in format\n    record.message = record.getMessage()\n_ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ \n\nself = <LogRecord: root, 20, /home/runner/work/pythonSelenium/pythonSelenium/tests/regression_ui/test_ui_regression.py, 154, \"An error occurred: \">\n\n    def getMessage(self):\n        \"\"\"\n        Return the message for this LogRecord.\n    \n        Return the message for this LogRecord after merging any user-supplied\n        arguments with the message.\n        \"\"\"\n        msg = str(self.msg)\n        if self.args:\n>           msg = msg % self.args\nE           TypeError: not all arguments converted during string formatting\n\n/opt/hostedtoolcache/Python/3.8.18/x64/lib/python3.8/logging/__init__.py:373: TypeError","steps":[{"name":"Log in to Trello account.","time":{"start":1734113668880,"stop":1734113686338,"duration":17458},"status":"passed","steps":[{"name":"Navigate to the Trello login page","time":{"start":1734113668880,"stop":1734113672197,"duration":3317},"status":"passed","steps":[],"attachments":[],"parameters":[],"stepsCount":0,"shouldDisplayMessage":false,"attachmentsCount":0,"hasContent":false,"attachmentStep":false},{"name":"Enter valid credentials (email, password, and 6-digits verification code)","time":{"start":1734113672197,"stop":1734113686338,"duration":14141},"status":"passed","steps":[],"attachments":[],"parameters":[],"stepsCount":0,"shouldDisplayMessage":false,"attachmentsCount":0,"hasContent":false,"attachmentStep":false}],"attachments":[],"parameters":[],"stepsCount":2,"shouldDisplayMessage":false,"attachmentsCount":0,"hasContent":true,"attachmentStep":false},{"name":"Open a board with an existing list and create a new card","time":{"start":1734113686338,"stop":1734113696685,"duration":10347},"status":"broken","statusMessage":"TypeError: not all arguments converted during string formatting\n","statusTrace":"  File \"/home/runner/work/pythonSelenium/pythonSelenium/tests/regression_ui/test_ui_regression.py\", line 154, in test_card_creation\n    log.info(\"An error occurred: \", str(e))\n  File \"/opt/hostedtoolcache/Python/3.8.18/x64/lib/python3.8/logging/__init__.py\", line 2082, in info\n    root.info(msg, *args, **kwargs)\n  File \"/opt/hostedtoolcache/Python/3.8.18/x64/lib/python3.8/logging/__init__.py\", line 1446, in info\n    self._log(INFO, msg, args, **kwargs)\n  File \"/opt/hostedtoolcache/Python/3.8.18/x64/lib/python3.8/logging/__init__.py\", line 1589, in _log\n    self.handle(record)\n  File \"/opt/hostedtoolcache/Python/3.8.18/x64/lib/python3.8/logging/__init__.py\", line 1599, in handle\n    self.callHandlers(record)\n  File \"/opt/hostedtoolcache/Python/3.8.18/x64/lib/python3.8/logging/__init__.py\", line 1661, in callHandlers\n    hdlr.handle(record)\n  File \"/opt/hostedtoolcache/Python/3.8.18/x64/lib/python3.8/logging/__init__.py\", line 954, in handle\n    self.emit(record)\n  File \"/opt/hostedtoolcache/Python/3.8.18/x64/lib/python3.8/site-packages/_pytest/logging.py\", line 386, in emit\n    super().emit(record)\n  File \"/opt/hostedtoolcache/Python/3.8.18/x64/lib/python3.8/logging/__init__.py\", line 1093, in emit\n    self.handleError(record)\n  File \"/opt/hostedtoolcache/Python/3.8.18/x64/lib/python3.8/logging/__init__.py\", line 1085, in emit\n    msg = self.format(record)\n  File \"/opt/hostedtoolcache/Python/3.8.18/x64/lib/python3.8/logging/__init__.py\", line 929, in format\n    return fmt.format(record)\n  File \"/opt/hostedtoolcache/Python/3.8.18/x64/lib/python3.8/site-packages/_pytest/logging.py\", line 139, in format\n    return super().format(record)\n  File \"/opt/hostedtoolcache/Python/3.8.18/x64/lib/python3.8/logging/__init__.py\", line 668, in format\n    record.message = record.getMessage()\n  File \"/opt/hostedtoolcache/Python/3.8.18/x64/lib/python3.8/logging/__init__.py\", line 373, in getMessage\n    msg = msg % self.args\n","steps":[],"attachments":[],"parameters":[],"stepsCount":0,"shouldDisplayMessage":true,"attachmentsCount":0,"hasContent":true,"attachmentStep":false}],"attachments":[{"uid":"8d0f142733d6b5ec","name":"log","source":"8d0f142733d6b5ec.txt","type":"text/plain","size":58344}],"parameters":[],"stepsCount":4,"shouldDisplayMessage":true,"attachmentsCount":1,"hasContent":true,"attachmentStep":false},"afterStages":[{"name":"driver::0","time":{"start":1734113696815,"stop":1734113696888,"duration":73},"status":"passed","steps":[],"attachments":[],"parameters":[],"stepsCount":0,"shouldDisplayMessage":false,"attachmentsCount":0,"hasContent":false,"attachmentStep":false}],"labels":[{"name":"tag","value":"TC004"},{"name":"tag","value":"TC000"},{"name":"parentSuite","value":"tests.regression_ui"},{"name":"suite","value":"test_ui_regression"},{"name":"subSuite","value":"TestUIRegression"},{"name":"host","value":"fv-az1756-422"},{"name":"thread","value":"2849-MainThread"},{"name":"framework","value":"pytest"},{"name":"language","value":"cpython3"},{"name":"package","value":"tests.regression_ui.test_ui_regression"},{"name":"resultFormat","value":"allure2"}],"parameters":[],"links":[],"hidden":false,"retry":false,"extra":{"severity":"normal","retries":[],"categories":[{"name":"Test defects","matchedStatuses":[],"flaky":false}],"history":{"statistic":{"failed":0,"broken":6,"skipped":0,"passed":18,"unknown":0,"total":24},"items":[{"uid":"c391704303e35e14","status":"passed","time":{"start":1734050827277,"stop":1734050860031,"duration":32754}},{"uid":"3e262912511e2920","status":"passed","time":{"start":1734027424002,"stop":1734027456872,"duration":32870}},{"uid":"20c7a57118fdea83","status":"passed","time":{"start":1733973794859,"stop":1733973827605,"duration":32746}},{"uid":"3923fab0d2e3e24f","status":"passed","time":{"start":1733972660597,"stop":1733972693477,"duration":32880}},{"uid":"f24c4a668e27b20e","status":"passed","time":{"start":1733971695095,"stop":1733971727842,"duration":32747}},{"uid":"31f3addecb4838f2","status":"passed","time":{"start":1733970440746,"stop":1733970473650,"duration":32904}},{"uid":"6986c570b25c4dcd","status":"passed","time":{"start":1733963339644,"stop":1733963372830,"duration":33186}},{"uid":"300e2d4d61674db6","status":"passed","time":{"start":1733932611358,"stop":1733932644711,"duration":33353}},{"uid":"5626296f44c9b828","status":"passed","time":{"start":1733891453809,"stop":1733891486519,"duration":32710}},{"uid":"392b601afedfe763","status":"passed","time":{"start":1733891009587,"stop":1733891042348,"duration":32761}},{"uid":"88b8625aef89bc7d","status":"broken","statusDetails":"TypeError: not all arguments converted during string formatting","time":{"start":1733890465322,"stop":1733890503647,"duration":38325}},{"uid":"e9ec09a5f05cf4ea","status":"broken","statusDetails":"TypeError: not all arguments converted during string formatting","time":{"start":1733889843558,"stop":1733889881879,"duration":38321}},{"uid":"b1375d4f74d28a48","status":"broken","statusDetails":"TypeError: not all arguments converted during string formatting","time":{"start":1733888700238,"stop":1733888738729,"duration":38491}},{"uid":"dbe0b17e2f869d8d","status":"passed","time":{"start":1733843222348,"stop":1733843255086,"duration":32738}},{"uid":"9fa7d1dc42d12fae","status":"broken","statusDetails":"TypeError: not all arguments converted during string formatting","time":{"start":1733842712495,"stop":1733842750891,"duration":38396}},{"uid":"bf395ad70de3111b","status":"broken","statusDetails":"TypeError: not all arguments converted during string formatting","time":{"start":1733841866105,"stop":1733841904338,"duration":38233}},{"uid":"427dfdead3e9b445","status":"passed","time":{"start":1733840260094,"stop":1733840292854,"duration":32760}},{"uid":"e47f88cec87f6072","status":"passed","time":{"start":1733797204126,"stop":1733797232419,"duration":28293}},{"uid":"20e98e3a890bf54a","status":"passed","time":{"start":1733586085542,"stop":1733586113888,"duration":28346}},{"uid":"2a080ef6a887df51","status":"passed","time":{"start":1733548536303,"stop":1733548564843,"duration":28540}}]},"tags":["TC004","TC000"]},"source":"a8d23136adcbda7d.json","parameterValues":[]}